application/src/main/java/org/togetherjava/tjbot/features/github/GitHubReference.java

```diff
@@ -181,6 +185,26 @@ MessageEmbed generateReply(GHIssue issue) throws UncheckedIOException {
         }
     }
 
+    /**
+     * Returns the state of the issue/PR
+     */
+    private int getIssueState(GHIssue issue) throws IOException {
+        if (issue instanceof GHPullRequest pR) {
```

Comment:
"pR" is not a good name for a variable, call it `pullRequest` or just `pr` instead